tools/mot_error_visualization.py

```diff
@@ -0,0 +1,253 @@
+import argparse
+import os
+import os.path as osp
+
+import cv2
+import mmcv
+import motmetrics as mm
+import numpy as np
+import seaborn as sns
+from mmcv import Config
+
+from mmtrack.datasets import build_dataset
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='visualize the situation of false positive, '
+        'false negative and ID switch for multiple object tracking')
+    parser.add_argument('config', help='test config file path')
+    parser.add_argument('--result-file', help='path of inference result')
+    parser.add_argument(
+        '--out-dir',
+        help='directory where painted images or videos will be saved')
+    parser.add_argument(
+        '--out-video', action='store_true', help='whether to output video')
+    parser.add_argument(
+        '--out-image', action='store_true', help='whether to output image')
+    parser.add_argument(
+        '--show',
+        action='store_true',
+        help='whether to show the results on the fly')
+    parser.add_argument(
+        '--fps', type=int, default=3, help='FPS of the output video')
+    args = parser.parse_args()
+    return args
+
+
+def show_wrong_tracks(img,
+                      bboxes,
+                      ids,
+                      wrong_types,
+                      bbox_colors=None,
+                      thickness=2,
+                      font_scale=0.4,
+                      text_width=10,
+                      text_height=15,
+                      show=False,
+                      wait_time=0,
+                      out_file=None):
+    """Show the wrong tracks with opencv.
+
+    Args:
+        img (str or ndarray): The image to be displayed.
+        bboxes (ndarray): A ndarray of shape (k, 5).
+        ids (ndarray): A ndarray of shape (k, ).
+        wrong_types (ndarray): A ndarray of shape (k, ),
+            where 0 denotes FP, 1 denotes FN and 2 denotes IDSW.
+        bbox_colors (list[tuple], optional): A list of colors.
+        thickness (int): Thickness of lines.
+        font_scale (float): the font scale of id and score.
+        text_width (int): the width of id and score.
+        text_height (int): the height of id and score.
+        show (bool): Whether to show the image on the fly.
+        wait_time (int): Value of waitKey param.
+        out_file (str, optional): The filename to write the image.
```

Comment:
for each argument with default value, please add `optional` after the type, and add `defaults to ***` at last. 
You can refer to [here](https://github.com/open-mmlab/mmtracking/blob/master/mmtrack/datasets/mot_challenge_dataset.py#L23)